app/helpers/status_helper.rb

```diff
@@ -1,28 +0,0 @@
-# Methods relating to inferring patient status
```

Comment:
This got moved into `app/models/concerns/statusable.rb` in the great patient-pregnancy merging.